scripts/performance/run-performace-tests-functions.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+set -eo pipefail
+source /var/tmp/functions.sh
+
+function usage() {
+    cat << HEREDOC
+Usage : $0 [--setup-client, --fetch-setup-info]
+where,
+    --setup-client - Setup S3 Client.
+    --fetch-setup-info - Fetch setup information for PerfPro.
+HEREDOC
+}
+
+ACTION="$1"
+if [ -z "$ACTION" ]; then
+    echo "ERROR : No option provided"
+    usage
+    exit 1
+fi
+
+function fetch_build_url() {
+    RGW_IMAGE=$(kubectl get pod $(kubectl get pods | awk '/cortx-server/{print $1; exit}') -o jsonpath="{.spec.containers[*].image}" | tr ' ' '\n' | uniq)
+    BUILD_ID=$(docker run --rm $RGW_IMAGE cat /RELEASE.INFO | grep BUILD | cut -d' ' -f2 | sed 's/"//g')
+    BUILD_URL="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/rockylinux-8.4/$BUILD_ID/prod/"
+}
+
+function create_endpoint_url() {
+    echo SOLUTION_FILE=$SOLUTION_FILE
+    ACCESS_KEY=$(yq e '.solution.common.s3.default_iam_users.auth_admin' $SOLUTION_FILE)
+    SECRET_KEY=$(yq e '.solution.secrets.content.s3_auth_admin_secret' $SOLUTION_FILE)
+    HTTP_PORT=$(kubectl get svc cortx-io-svc-0 -o=jsonpath='{.spec.ports[?(@.port==80)].nodePort}')
+    IP_ADDRESS=$(ifconfig eth1 | grep inet -w | awk '{print $2}')
+    ENDPOINT_URL="http://$IP_ADDRESS:$HTTP_PORT"
+
+    echo ENDPOINT_URL=$ENDPOINT_URL
+    echo ACCESS_KEY=$ACCESS_KEY
+    echo SECRET_KEY=$SECRET_KEY
+    echo BUILD_URL=$BUILD_URL
+}
+
+function run_io_sanity() {
```

Comment:
can we add this function in functions.sh as other io sanity jobs are also using same code?

Reply:
Fixed it